bin/cron/process_forms

```diff
@@ -68,6 +68,9 @@ def process_batch_of_forms
         notified_at: DATE_FOR_ERRORS
       )
       next
+    rescue Exception => e
+      ChatClient.log "Unable to process form #{form[:id]} because #{e.message}."
+      raise e
```

Comment:
Note that we could expand the `rescue AbortFormError` to also include parse errors or otherwise. It isn't clear to me whether this is a change we want. Let me know if it is, I'm happy to do so as part of this PR.